content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -83,24 +78,127 @@ Resource associated with Activity:
     service.instance.id: 20c891c2-94b4-4203-a960-93a22e837a32
 ```
 
+</details>
+
 This output matches the span created in the preceding code sample.
 
+### Metrics
+
+Next, add metrics to the tracing code. This will initialize a
+[Meter](/docs/concepts/signals/metrics), and will use the Meter to create a
+Counter.
+
+```csharp
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+// Define some important constants to initialize tracing with
+var serviceName = "MyCompany.MyProduct.MyService";
+var serviceVersion = "1.0.0";
+
+// Factor out the ResourceBuilder so it can be shared with the tracer provider
+// and the meter provider.
+var appResourceBuilder = 
+    ResourceBuilder.CreateDefault()
+        .AddService(serviceName: serviceName, serviceVersion: serviceVersion);
+
+var MyActivitySource = new ActivitySource(serviceName);
+
+// Configure important OpenTelemetry settings and the console exporter
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
```

Comment:
Hmmmm. Alarm bells are going off in my head about people putting tracing initialization into helper methods. Do you have an example where someone got confused here? A potential change here could be to write a helper that just returns the `TracerProvider` like so?

```csharp
TracerProvider ConfigureTracerProvider(...)
{
    return Sdk.CreateTracerProvider(). [... everything else...]
}

using var tracerProvider = ConfigureTracerProvider(...)
```

I'd generally prefer to keep the `using var`, but if it's too confusing we can remove it. My main issue with doing that would be needing developers to then also keep track of the fact that _somewhere else_ there needs to be a call to dispose, and that _somewhere else_ could be hard to remember or even notice.

Reply:
We have seen a lot of poorly written code (by copy-pasting the code and not removing "using"). That led to this PR:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2848/files 

:) 


```
void EnableOpenTelemetry(...)
{
    using provider = sdk....; // copy pasted content here..
}

// startup
EnableOpenTelemetry();

```

Reply:
Hmmmm. I wonder...what if we just remove the console app from this page? Since web is the most common scenario, we could just do that (this is what python does with flask). And then in the manual instrumentation page, we can elaborate a lot more about the lifecycle of a TracerProvider or a MeterProvider.

Reply:
Another alternative is to elaborate in this document, although that could get a little lengthy and people might skip it since they're just looking for an easy way to get started.

Reply:
@cijothomas any thoughts here?

Reply:
I'll remove the console section from this page for now and figure out a good place to document use of the API for non-aspnet scenarios outside of getting-started.

Reply:
Filed https://github.com/open-telemetry/opentelemetry.io/issues/2048 to track documenting this better

Reply:
Yep, let's do that for now. This at least eliminates the main problem, even if it's not solved in the ideal way.